Project.toml

```diff
@@ -4,6 +4,8 @@ author = "Júlio Hoffimann <julio.hoffimann@gmail.com>"
 version = "0.16.5"
 
 [deps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+ComputationalResources = "ed09eef8-17a6-5b46-8889-db040fac31e3"
```

Comment:
Are you adding these lines manually? You should use the package manager instead because it orders dependencies in lexicographic order. Please don't edit Project.toml and Manifest.toml manually, except when you need to adjust a specific version of the deps. BTW, the versions of the dependencies used should be listed in the [compat] section.

Reply:
I didn't add those lines manually. I used the package manager. I did not edit neither Project.toml nor Manifest.toml manually.

Reply:
Anything still to do here?